packages/core/rendering/core-rendering-server-internal/src/index.ts

```diff
@@ -7,6 +7,7 @@
  */
 
 export { RenderingService } from './rendering_service';
+export { Fonts } from './views';
```

Comment:
TIL that the `security` plugin was using this internal react component (see `x-pack/plugins/security/server/prompt_page.tsx`).

If at some point we have more usages of our internal components related to the page templating we should think about extracting them to a public utility package, but given there was one single usage of it, I just adapted the import to point to `@kbn/core-rendering-server-internal` (and had to export the component from the entrypoint)

Reply:
The whole reason we have internal components is to allow for changes that don't affect upstream consumers. At this stage, I don't think it's possible to enforce that, and we'll have to adapt as the need arises.